concepticondata/conceptlists/Duong-2020-100.tsv

```diff
@@ -0,0 +1,101 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_GLOSS	CONCEPTICON_ID
+Duong-2020-100-1	1	sun	SUN	1343
+Duong-2020-100-2	2	moon	MOON	1313
+Duong-2020-100-3	3	star	STAR	1430
+Duong-2020-100-4	4	earth	EARTH (SOIL)	1228
+Duong-2020-100-5	5	cloud	CLOUD	1489
+Duong-2020-100-6	6	rain	RAINING OR RAIN	2108
+Duong-2020-100-7	7	water	WATER	948
+Duong-2020-100-8	8	mountain	MOUNTAIN	639
+Duong-2020-100-9	9	rock	ROCK	668
+Duong-2020-100-10	10	sand	SAND	671
+Duong-2020-100-11	11	road/way	ROAD	667
+Duong-2020-100-12	12	fire	FIRE	221
+Duong-2020-100-13	13	smoke	SMOKE (EXHAUST)	778
+Duong-2020-100-14	14	ash	ASH	646
+Duong-2020-100-15	15	night	NIGHT	1233
+Duong-2020-100-16	16	tree	TREE	906
+Duong-2020-100-17	17	seed	SEED	714
+Duong-2020-100-18	18	leaf	LEAF	628
+Duong-2020-100-19	19	root	ROOT	670
+Duong-2020-100-20	20	bark	BARK	1204
```

Comment:
Now that's an unfortunate ordering to have tree bark followed by dog. Let's hope it didn't confuse the elicitors.

Reply:
:-) yes! I think we'll see from the lexical data, as they tend to use a compound with tree for tree bark.